pootle/static/js/welcome/components/RandomSymbolsBackground.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) Zing contributors.
+ *
+ * This file is a part of the Zing project. It is distributed under the GPL3
+ * or later license. See the LICENSE file for a copy of the license and the
+ * AUTHORS file for copyright and authorship information.
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import './RandomSymbolsBackground.css';
+
+// Note that depending on your IDE/Editor,
+// some symbols in the string below may appear as blank
+// or tofu ones; they are in fact legitimate different
+// symbols that modern browsers can render just fine
+const symbols =
+  'ABCFGHJKQRSTWXYZjmgpmĂĘŠŸƵǮбБГДёжЙФЩЪЮЯ' +
+  'ÇØßÆÄÃÛÜÐÑΣΘΞΨΩΪαβγωϢϪξԹՃשאتيऄऴআঞਆੴ' +
+  'ઊணஜఱఫฬง༖ဋႴႵᏤᎺぁじぢツポ㑈㓗㒲겅걚';
+
+const numColors = 4;
+const numChars = 200;
+const fontSizeVariation = 20; // px
+const minFontSize = 30; // px
+const opacityVariation = 0.3; // 0..1
+const minOpacity = 0.2; // 0..1
+const blurVariation = 5; // 0, 1, 2, ...
+const minBlur = 2; // 0, 1, 2, ...
+
+
+class RandomSymbolsBackground extends React.Component {
+
+  static propTypes() {
+    return {
+      items: React.PropTypes.array.isRequired,
+    };
+  }
+
+  static shouldComponentUpdate() {
+    return false;
+  }
+
+  componentDidMount() {
```

Comment:
No much reason. Updated in 441013ee7c4d15152a762144552800b05d61e2f4. Now that I'm thinking of it, should we probably merge `componentWillMount()` and `render()` [in RandomMessage.js](https://github.com/evernote/zing/blob/d68c79f42454eb713a699967f28ded108450f617/pootle/static/js/welcome/components/RandomMessage.js#L26-L44)?